### PR TITLE
proxy: Add various debug logging

### PR DESCRIPTION
**Pull request by cgwalters**
To aid debugging https://github.com/ostreedev/ostree-rs-ext/issues/657

### CODE REVIEW

File: cmd/skopeo/proxy.go [hunk 1]
```diff
@@ -600,6 +600,7 @@ func (h *proxyHandler) GetBlob(args []any) (replyBuf, error) {
 		tr := io.TeeReader(blobr, verifier)
 		n, err := io.Copy(f.w, tr)
 		if err != nil {
+			logrus.Debugf("")
```
Comment (mtrmac) [inline cmd/skopeo/proxy.go:603]: Do you want to add some distinct text here?

**Review by mtrmac — COMMENTED** [review]
Sure.

Feel free to merge as is this is sufficient.

File: cmd/skopeo/proxy.go (hunk 1 above)
Comment (cgwalters) [reply cmd/skopeo/proxy.go]: Forgot to amend the commit after vacillating on whether a debug log is needed there; I don't think it is, so deleted.

File: cmd/skopeo/proxy.go [hunk 2]
```diff
@@ -782,6 +784,8 @@ func (h *proxyHandler) processRequest(readBytes []byte) (rb replyBuf, terminate
 		err = fmt.Errorf("invalid request: %v", err)
 		return
 	}
+	logrus.Debugf("Executing method %s", req.Method)
```
Comment (mtrmac) [inline cmd/skopeo/proxy.go:787]: (For very broken or adversarial values, `%q` throughout would be better, but that’s not really relevant here.)

**Review by mtrmac — APPROVED** [review]
Thanks!

File: cmd/skopeo/proxy.go (hunk 2 above)
Comment (cgwalters) [reply cmd/skopeo/proxy.go:787]: Yeah...I guess I need to just internalize "use %q by default" in general.

**Comment by mtrmac** [comment]
@cgwalters please try rebasing on top of `main`; that will include an updated `golangci-lint`, and there’s a _chance_ that might affect the `osx` test timing out. At least, we would be working off a shared baseline.

**Pull request closed without merging**